zerver/tests/test_markdown.py

```diff
@@ -1817,7 +1817,7 @@ def test_mention_wildcard(self) -> None:
             rendering_result.rendered_content,
             '<p><span class="user-mention" data-user-id="*">@all</span> test</p>',
         )
-        self.assertTrue(rendering_result.mentions_wildcard)
+        self.assertTrue(rendering_result.mentions_stream_wildcard)
 
```

Comment:
Added a [prep commit.](https://github.com/zulip/zulip/pull/25828/commits/3e1c48fe2f158ec47c058ce1a8fbe73ee3348659)

I have kept it after the rename commit to avoid using `wildcard` then again correcting it to `stream_wildcard`. Since we are introducing a new function, it makes more sense to add it with the `stream_wildcard` term.